Extensions/FileSystem/JsExtension.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ * 
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ * 
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it. 
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ * 
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+  createExtension: function (t, gd) {
+    const extension = new gd.PlatformExtension();
+    extension.setExtensionInformation(
+      "FileSystem",
+      t("Filesystem"),
+      t(
+        "Access the filesystem of the operating system."
+      ),
+      "Matthias Meike",
+      "Open source (MIT License)"
+    ).setExtensionHelpPath("/all-features/filesystem");
+
+    extension
+      .addStrExpression(
+        "SpecialFolderDesktop",
```

Comment:
I think we can name them simply `DesktopPath`. I think Special does not have any value here (it's already "special" int the sense that we're giving to the user a method specially for it ;)) and as mentionned in another comment, I think Path would be more consistent than Folder with other programming languages/libraries.

(Expressions will be prefixed by the extension name, so we'll have `FileSystem::DesktopPath()` which looks good - or `FileSystem::GetDesktopPath()` if you prefer - but as there is no parameter we can think of DesktopPath almost like a constant so the "Get" could be skipped from the name).